ocs_ci/utility/utils.py

```diff
@@ -643,20 +645,17 @@ def exec_cmd(
         stderr     (str): The standard error (None if not captured).
 
     """
-    masked_cmd = mask_secrets(cmd, secrets)
-    log.info(f"Executing command: {masked_cmd}")
+    _env = kwargs.pop("env", os.environ.copy())
```

Comment:
How do you think, will this brake logic when "--kubeconfig" is provided in string of the command? 
see: https://github.com/red-hat-storage/ocs-ci/blob/c3536dfc7fb98fef4b0d042eb5c0031c24dd8bb6/ocs_ci/deployment/hosted_cluster.py#L685-L710

Reply:
no `--kubeconfig` cli argument has precedence - but in case we do not provide, I still want to be sure we have KUBECONFIG also in env variable just to be on safe side in case some other command we used in the past relied on env var from some reason.

Before we had anyway KUBECONFIG exported in whole framework process - so we always had it exported